wpeverest/everest-forms
# Enhance - Dashboard page

### All Submissions:



### Changes proposed in this Pull Request:

Previously, there was only activate button to activate the addon but no place to enter the license key. This PR adds the feature to enter the plugin license key to activate the addons. It also adds the feature to play the demo video after activating the license. It also adds the feature to search the modules.

Closes # .

### How to test the changes in this Pull Request:

1. Activate both Everest Forms and Everest Forms Pro
2. But do not activate the license key on the Pro version
3. Go to Dashboard page and Click on **Upgrade Plan** button.
4. A pop up to enter the license key must appear to activate the license.
5. After license activation. Toggle button is used to activate/ deactivate the features.
6. The settings button must appear to those features which has the global settings.
7. Each settings button must redirect to respective global settings page.
8. Now, go to Help page and click on the copy button to copy the respective shortcodes.

### Types of changes:



### Other information:



### Changelog entry

> Enhance - Dashboard page
